docs/gaia/delegator-guide-cli.md

```diff
@@ -336,12 +336,16 @@ For each command, you can use the `-h` or `--help` flag to get more information.
 
 ## Sending Transactions
 
+::: warning
+On Cosmos Hub mainnet, the accepted denom is `uatom`, where `1atom = 1.000.000uatom`
```

Comment:
I believe using `.` to separate thousands is a european thing. Do we want to do that here, or will it be confusing?